server/routes/incentiveLevels.ts

```diff
@@ -284,3 +295,29 @@ export default function routes(router: Router): Router {
 
   return router
 }
+
+async function errorMessageWhenCannotDeactivate(prisonApi: PrisonApi, errorResponse: ErrorResponse): Promise<string> {
+  let message = 'This level cannot be made inactive because some establishments are using it.'
+
+  const prisonNames: string[] = []
+  const prisonIds: string[] | undefined = (errorResponse?.moreInfo ?? '')
+    .split(',')
+    .map(prisonId => prisonId.trim())
+    .filter(prisonName => prisonName)
+  // eslint-disable-next-line no-restricted-syntax
+  for (const prisonId of prisonIds) {
```

Comment:
Probably not much of a difference in practice as I'd hope they'd only try to remove a level only when most of the prisons already stopped using it...

But we could potentially make all these requests at once concurrently as retrieving the information for each of the prisons is independent from the others.

Reply:
hmm not a bad idea!

Reply:
how about this? 0c2a5c2e56fef6c7738e61870b99afec7f9c0419